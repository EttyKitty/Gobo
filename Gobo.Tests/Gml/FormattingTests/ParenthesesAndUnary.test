foo++
bar
--
foo

++bar

baz
--qux

x = !(!(!(a)))
y = (((--(+a))++))
z = (((--(+a))++))()

// '!' should be outside
cond = (not(condition || bar && (baz)))

// '!' should be inside
if (not(condition || bar && (+(baz)))) {}

var (unwrap_me) = 1;
static ((unwrap_me_twice)) = 2;

try {} catch ((bar)) {}