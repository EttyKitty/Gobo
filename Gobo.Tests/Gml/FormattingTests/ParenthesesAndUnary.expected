foo++;
bar;
--foo;

++bar;

baz;
--qux;

x = !(!(!a));
y = (--(+a))++;
z = ((--(+a))++)();

// '!' should be outside
cond = !(condition || bar && baz);

// '!' should be inside
if (!(condition || bar && +baz)) {}
